content/en/docs/concepts/architecture/mixed-version-proxy.md

```diff
@@ -58,9 +58,9 @@ these fields in the API server configuration file.
 If these flags are unspecified, peers will use the value from either `--advertise-address` or
 `--bind-address` command line argument to the kube-apiserver. If those too, are unset, the host's default interface is used.
 
-## Mixed version proxying
+## Mixed Version Proxying
```

Comment:
All headings in that file should be in title case, and Mixed Version Proxy should be mixed version proxy (italicized on first use).

Reply:
Done